src/HID.cc

```diff
@@ -456,23 +425,15 @@ NAN_METHOD(HID::write)
     return Nan::ThrowError("HID write requires one argument");
   }
 
-  try {
-    HID* hid = Nan::ObjectWrap::Unwrap<HID>(info.This());
+  HID* hid = Nan::ObjectWrap::Unwrap<HID>(info.This());
```

Comment:
It is possible that removing the try/catch here was a bit optimistic since it could be possible to have the Unwrap or ToObject() fail. On the other hand, the only code that has access to these native calls is the JS facade.